modules/@apostrophecms/page-type/index.js

```diff
@@ -215,6 +215,49 @@ module.exports = {
   },
   methods(self) {
     return {
+      // Given a page and its parent (if any), returns a schema that  is
+      // filtered appropriately to that page's type, taking into account whether
+      // the page is new, whether it is parked, and the parent's allowed subpage
+      // types.
+      allowedSchema(req, page = {}, parentPage = {}) {
+        let schema = self.schema;
```

Comment:
By not calling the super version of this method you're losing the permissions-related filtering that happens there. You still need to do that to get your initial schema for further processing.

Reply:
So you should be using extendMethods here.

Reply:
my bad. I forgot my own plan.